src/journal/journald.c

```diff
@@ -52,7 +52,7 @@ int main(int argc, char *argv[]) {
                 goto finish;
 
         server_vacuum(&server, false);
-        server_flush_to_var(&server);
+        server_flush_to_var(&server, true);
```

Comment:
Summary: this PR helps to fix #1397, but it doesn't fix #1397 :-)

[This](https://github.com/systemd/systemd/pull/4879#discussion_r92835528)

> To provide a nice way for this I figure we could "journalctl --make-persistent" or so which creates the directory under a temporary name first, then applies the right ACLs and then moves it the final name and sends SIGUSR1 to journald to make it flush its stuff there. I think that would work fine and solve the points you raise, wouldn't it?

fixes #1397